ru/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: news_post
+title: "Support for Ruby 1.9.3 has ended"
+author: "Olivier Lacan"
+translator: "arbox"
+date: 2015-02-23 00:00:00 +0000
+lang: ru
+---
+
+С сегодняшнего дня любая поддержка Ruby версии 1.9.3 закончена. Исправления
+ошибок и проблем безопасности больше не будут портироваться из более старших
+версий в версию 1.9.3.
+
+Конец подержки [был анонсирован больше года назад
```

Comment:
Strange, I thought it is the correct way to break out links in Markdown... Done!